queries.sql

```diff
@@ -8,3 +8,41 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg  BETWEEN 10.4 and 17.3;
+
+BEGIN;
+SAVEPOINT save1;
+DELETE FROM animals;
+ROLLBACK TO SAVEPOINT save1;
+COMMIT;
+
```

Comment:
Great job @flemton 🤝

- [ ] As stated in the project requirements, Inside a transaction update the `animals` table by setting the species column to unspecified. Verify that the change was made. Then roll back the change and verify that the species columns went back to the state before the transaction. Kindly add this transaction to the `queries.sql` file.